src/components/map/MapService.js

```diff
@@ -784,6 +830,7 @@ goog.require('ga_urlutils_service');
                 useInterimTilesOnError: gaNetworkStatus.offline,
                 extent: extent
               });
+              loadsuccess = 'tileloadend';
```

Comment:
Can't  we have function taking  an olLayer as parameter wichich returns an object with all events :+1: 
```
function  getTIleLoadEvents(olLayer) {
    .
    .
    .
    returns {
      loadError:
      loadSuccess:
      load:   
    }
}

```